nss-day-cohort-47/reactive-nutshell-the-fantastic-404s
# Lm article images

# Description
Fixing form for articles (date on edit/new) and adding an image to the form, which will appear in the highlight.

## Type of change
Fixes form and adding new feature (image)

## Testing Instructions
Please run natively and make sure an image is appearing on the dashboard and the article tab form can now include an image. I will send you updated data in Slack for json that includes images.

## Checklist:
[x ] My code follows the style guidelines of this project
[x ] I have performed a self-review of my own code
[ ] I have commented my code, particularly in hard-to-understand areas
[x ] My changes generate no new warnings or errors
[x ] I have added test instructions that prove my fix is effective or that my feature is working
